test/toucan2/jdbc/row_test.clj

```diff
@@ -32,9 +32,6 @@
           conj
           (select/reducible-select ::test/venues 1)))))
 
-(defn- realized-keys [^TransientRow row]
-  @(.realized_keys row))
```

Comment:
Couldn't this still be calculated by looking at `.column_names` and removing that ones that are `::not-found`? That way we don't need to delete a ton of test code below